### DISCUSSION TITLE
Gist search not possible anymore

### DISCUSSION THREAD

**Discussion by madhums** [community/community#140427] [Repositories] [answered]
Gist search used to display a dropdown of results when I entered some text from my own gists. Now it is impossible to search my own gists. Is there some alternative to click every page out of hundreds to find something I am looking for in my own gist?

**Comment by dotspencer** [comment, marked as answer]
Very frustrating that this feature was removed. 

You can still lookup your gists using for example `user:dotspencer python` to find my gists with "python" but it's such a pain compared to how simple it was before.

**Reply by madhums** [reply]
Yea indeed, but glad this works although a bit more painfully.

**Comment by mattheworiordan** [comment]
Hit the same wall. 500 gists over the years and no way to find anything. GitHub search doesn't index gists, and `gh gist list --include-content --filter` is ~30s on my account, because it's serial API calls.

I scratched it: https://github.com/mattheworiordan/gistgrep -> Local mirror, incremental sync, fzf TUI with ranked previews, and on-device AI summaries via Apple Intelligence.

*Search is now sub-100ms once synced*

<img width="1065" height="425" alt="Image" src="https://github.com/user-attachments/assets/7b56257a-b4c2-461c-8615-901b54421b32" />

`brew install mattheworiordan/tap/gistgrep`. macOS only right now (the summary bit uses FoundationModels). Happy to add a Linux port via Ollama if there's interest, please do raise an issue. I hope my little project helps!
